src/components/ProjectCard/Project.js

```diff
@@ -11,6 +11,8 @@ const Project = props => {
   const { project } = props
   const { display_image, name, description, hosted_url, category, is_approved } = project
 
+  console.log(category)
```

Comment:
Only YOU can prevent random console logs 🚭

Reply:
Removed!